source/common/util/ignore-path.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * BEGIN HEADER
+ *
+ * Contains:        Utility function
+ * CVM-Role:        <none>
+ * Maintainer:      Hendrik Erz
+ * License:         GNU GPL v3
+ *
+ * Description:     Utility functions to check for ignored file paths.
+ *
+ *
+ * END HEADER
+ */
+
+// A list of path patterns that should be ignored
+// by the FSAL layer and Chokidar watchdog process
+const IGNORE_PATH_RE: RegExp[] = [
+  /(?:^|[\/\\])\.DS_Store$/i, // macOS directory files
```

Comment:
Can we go back to constructing a list of regexp's like before? This would make adapting the non-changing part of the regexps simpler, and possibly make the list of file types we ignore easier to ignore, too?